src/main/java/com/library/java_library_management/controller/Controller.java

```diff
@@ -0,0 +1,52 @@
+package com.library.java_library_management.controller;
+
+import com.library.java_library_management.repository.TestModeRepository;
+import com.library.java_library_management.repository.Repository;
+import com.library.java_library_management.service.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Controller {
+    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+    private Service service;
```

Comment:
```suggestion
    private final Service service;
```

의존성 주입 시 final 키워드를 사용하여 해당 클래스가 변경이 일어나지 않음을 명시하는 것이 좋습니다.